src/ui/zcl_abapgit_gui_page_debuginfo.clas.abap

```diff
@@ -161,6 +162,8 @@ CLASS zcl_abapgit_gui_page_debuginfo IMPLEMENTATION.
       iv_typ = zif_abapgit_html=>c_action_type-url ).
     ri_html->add( '</div>' ).
 
+    SELECT SINGLE extrelease FROM cvers INTO lv_sp WHERE component = 'SAP_BASIS' ##SUBRC_OK.
```

Comment:
the `SELECT` seems a bit malplaced here, perhaps move it to a method in `zcl_abapgit_factory=>get_environment( )` ?